src/components/sw360/Table/wrapper.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Helio Chissini de Castro, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { h, createRef as gCreateRef, Component as gComponent } from 'gridjs'
+import ReactDOM from 'react-dom'
+
+export class ReactWrapper extends gComponent<{
```

Comment:
Use Function Component instead of Class Component

Reply:
Same as above